int32               lift_state
geometry_msgs/Point uwb_pos
float32[11]         lidar_ranges
